packages/react-components/eslint-plugin-react-components/tsconfig.lib.json

```
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false,
+    "lib": ["ES2019"],
+    "declaration": true,
+    "declarationDir": "../../../dist/out-tsc/types",
+    "outDir": "../../../dist/out-tsc",
+    "inlineSources": true,
+    "types": ["static-assets", "environment", "node"],
+    "module": "CommonJS"
+  },
+  "exclude": ["**/*.spec.ts", "**/*.spec.tsx", "**/*.test.ts", "**/*.test.tsx", "src/index.dev.d.ts"],
```

you copied this from our storybook plugin ? :D 

```suggestion
  "exclude": ["**/*.spec.ts", "**/*.spec.tsx", "**/*.test.ts", "**/*.test.tsx"],
```

---

from babel-preset, actually 😄  I tried nx generators first, but we it's tied to workspace library generator meant to be used to generate react-component-lib with a lot of stuff like `library/stories`, etc. Original `@nx/js` library generator also generates a bit different from existing packages output.

---

yeah we need to implement a generic set of generators for these use-cases. it wasn't a priority till now but as we gonna grow our tooling it will be necessary to have. ty